### ISSUE TITLE
Voice Unlocker Error

### ISSUE THREAD

**Issue by johnysandels** [karashiiro/TextToTalk#52]
Error when trying to use the built in Voice unlocker, when clicking both `Manual tutorial` and also `Enable all system voices` on version 1.9.4 and 1.9.7. Other versions not tested.

There is nothing showing an issue in output or dalamud logs

![image](https://user-images.githubusercontent.com/79720646/141060884-d5ed33e0-df70-42c2-bded-756387b2022a.png)

**Comment by johnysandels** [comment]
![image](https://user-images.githubusercontent.com/79720646/141232783-6fcd6248-0296-4450-9f1c-7dd35b56e6ff.png)
This is what is in the dalamud log 

```
2021-11-10 19:45:04.320 -08:00 [ERR] [TextToTalk] VoiceUnlocker failed to start.
System.ComponentModel.Win32Exception (740): The requested operation requires elevation.
   at System.Diagnostics.Process.StartWithCreateProcess(ProcessStartInfo startInfo)
   at System.Diagnostics.Process.Start()
   at System.Diagnostics.Process.Start(ProcessStartInfo startInfo)
   at TextToTalk.VoiceUnlockerRunner.Execute() in K:\arashiiro\VoiceUnlockerRunner.cs:line 15
2021-11-10 19:45:04.334 -08:00 [INF] LASTEXCEPTION:eyJXaGVuIjoiMjAyMS0xMS0xMFQxOTo0NTowNC4zMzA3ODk3LTA4OjAwIiwiSW5mbyI6IlN5c3RlbS5Db21wb25lbnRNb2RlbC5XaW4zMkV4Y2VwdGlvbiAoNzQwKTogVGhlIHJlcXVlc3RlZCBvcGVyYXRpb24gcmVxdWlyZXMgZWxldmF0aW9uLlxyXG4gICBhdCBTeXN0ZW0uRGlhZ25vc3RpY3MuUHJvY2Vzcy5TdGFydFdpdGhDcmVhdGVQcm9jZXNzKFByb2Nlc3NTdGFydEluZm8gc3RhcnRJbmZvKVxyXG4gICBhdCBTeXN0ZW0uRGlhZ25vc3RpY3MuUHJvY2Vzcy5TdGFydCgpXHJcbiAgIGF0IFN5c3RlbS5EaWFnbm9zdGljcy5Qcm9jZXNzLlN0YXJ0KFByb2Nlc3NTdGFydEluZm8gc3RhcnRJbmZvKVxyXG4gICBhdCBUZXh0VG9UYWxrLlZvaWNlVW5sb2NrZXJSdW5uZXIuRXhlY3V0ZSgpIGluIEs6XFxhcmFzaGlpcm9cXFZvaWNlVW5sb2NrZXJSdW5uZXIuY3M6bGluZSAxNSIsIkNvbnRleHQiOiJbVGV4dFRvVGFsa10gVm9pY2VVbmxvY2tlciBmYWlsZWQgdG8gc3RhcnQuXG5TeXN0ZW0uQ29tcG9uZW50TW9kZWwuV2luMzJFeGNlcHRpb24gKDc0MCk6IFRoZSByZXF1ZXN0ZWQgb3BlcmF0aW9uIHJlcXVpcmVzIGVsZXZhdGlvbi5cclxuICAgYXQgU3lzdGVtLkRpYWdub3N0aWNzLlByb2Nlc3MuU3RhcnRXaXRoQ3JlYXRlUHJvY2VzcyhQcm9jZXNzU3RhcnRJbmZvIHN0YXJ0SW5mbylcclxuICAgYXQgU3lzdGVtLkRpYWdub3N0aWNzLlByb2Nlc3MuU3RhcnQoKVxyXG4gICBhdCBTeXN0ZW0uRGlhZ25vc3RpY3MuUHJvY2Vzcy5TdGFydChQcm9jZXNzU3RhcnRJbmZvIHN0YXJ0SW5mbylcclxuICAgYXQgVGV4dFRvVGFsay5Wb2ljZVVubG9ja2VyUnVubmVyLkV4ZWN1dGUoKSBpbiBLOlxcYXJhc2hpaXJvXFxWb2ljZVVubG9ja2VyUnVubmVyLmNzOmxpbmUgMTUifQ==
```

**Comment by karashiiro** [comment]
Should be fixed with v1.9.9.

**Comment by johnysandels** [comment]
It says it's successful on restart, but it doesn't seem to actually unlock all the voices.

**Comment by karashiiro** [comment]
Are you on Windows 10 or 11? I'm testing it myself right now on Windows 10 and it seems to work perfectly fine.

**Comment by johnysandels** [comment]
I'm testing on windows 10. Let me try again

**Comment by johnysandels** [comment]
I've reinstalled the voices and the voice unlocker says it worked, but on restart no new voices are listed.

**Comment by johnysandels** [comment]
Still can't find a resolution to this one unfortunately. Has anyone had luck with enabling all voices more recently? or is anyone willing to test by removing the voiced and trying to re-add them?

**Comment by johnysandels** [comment]
I just found an old version of the voice unlocker for version 1.8.6 and that old version worked, and now all of my voices are unlocked!
https://github.com/karashiiro/DalamudPlugins/commit/403fc68a9c76f2455db8811bb20e5d9b77d79771

**Comment by karashiiro** [comment]
https://raw.githubusercontent.com/karashiiro/TextToTalk/main/src/VoiceUnlocker/Program.cs

https://raw.githubusercontent.com/karashiiro/TextToTalk/62f769bd74511e923bee49427a89e9ee131fad33/VoiceUnlocker/Program.cs

The VoiceUnlocker code hasn't changed between 1.8.6 and now, so I'm not sure how that fixed it.

**Comment by karashiiro** [comment]
I guess the underlying framework changed (.NET Framework 4.7.2 to .NET 5), so I can try reverting that? Seems unlikely, though.

**Comment by karashiiro** [comment]
Released a testing build (v1.9.12.1) for anyone who who's still experiencing this problem to try.

**Comment by johnysandels** [comment]
Just tested this update on my partner's PC, and it worked perfectly! All of her voices are unlocked now! Weird that NET5 would somehow mess with the code 🥴

**Comment by karashiiro** [comment]
Thanks for testing this! I'll push this to the main release stream and close this, then 👏 Compatibility is always a major headache, I guess.
